docs/user/alerting/alerting-getting-started.asciidoc

```diff
@@ -92,22 +92,6 @@ Using the server monitoring example, each server with average CPU > 0.9 is track
 
 image::images/alerts.svg[{kib} tracks each detected condition as an alert and takes action on each alert]
 
-[float]
```

Comment:
Moved to the `Defining rules` page, to be closer to where the setup for this concept is.